libpod/runtime.go

```diff
@@ -293,8 +293,20 @@ func makeRuntime(runtime *Runtime) error {
 	if err != nil {
 		return err
 	}
+
 	runtime.store = store
 	is.Transport.SetStore(store)
+
+	// Set up image runtime and store in runtime
+	ir := image.NewImageRuntimeFromStore(runtime.store)
+	if err != nil {
+		return err
+	}
+
+	runtime.imageRuntime = ir
+
+	// Setting signaturepolicypath
+	ir.SignaturePolicyPath = RuntimeConfig{}.SignaturePolicyPath
```

Comment:
Shoudln't this be runtime.Config.SignaturePolicyPath?

Reply:
ok